lib/components/post-summary/post-summary.a11y.test.ts

```diff
@@ -0,0 +1,183 @@
+import { html } from "@open-wc/testing";
+import { runA11yTests } from "../../test/a11y-test-utils";
+import { getChildren } from "../../test/post-summary-test-utils";
+import type {
+    BadgeType,
+    TruncationSizes,
+} from "../../test/post-summary-test-utils";
+import "../../index";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const template = ({ component, testid }: any) => html`
```

Comment:
My overall impression is that there is a lot of repetition between the visual and a11y test files and more things can be moved in a setup file like the test utils you have created.

Reply:
💯 This is something I've been thinking about for our test broadly. Most of the a11y and visual tests for a given component overlap > 80%. I'll probably want to discuss a uniform way to do this across our components.

For now, I'll port much of the post-summary tests into the test utils and I'll create a ticket to investigate creating a pattern for merging overlapping test code.

@giamir If you think the pattern of a test-utils file that is imported into the a11y/visual test files is sufficient and doesn't require more consideration, please say so and I'll close the ticket.

Reply:
I think we can leave the ticket. In the meantime the test-utils is probably good enough (and maybe we won't even be able to come up with something simpler than that).